tests/bitmap_tests.rs

```diff
@@ -3,30 +3,30 @@ mod bitmap_tests {
     use ark_msm::bitmap::Bitmap;
 
     fn setup() -> Bitmap {
-        return Bitmap::new(67);
+        Bitmap::new(67)
     }
 
     #[test]
     fn test_set_when_only_once_return_false() {
         let mut bitmap = setup();
-        assert_eq!(bitmap.test_and_set(0), false);
-        assert_eq!(bitmap.test_and_set(33), false);
+        assert!(!bitmap.test_and_set(0));
+        assert!(!bitmap.test_and_set(33));
     }
 
     #[test]
     fn test_set_when_more_than_once_return_true() {
         let mut bitmap = setup();
-        assert_eq!(bitmap.test_and_set(0), false);
-        assert_eq!(bitmap.test_and_set(0), true);
+        assert!(!bitmap.test_and_set(0));
+        assert!(bitmap.test_and_set(0));
```

Comment:
Changes here make the code shorter, but not necessarily easier to read. "true" and "false" looks good here.

Reply:
Comparing `true` and `false` via `assert_eq` is bad form. There are no manual changes here, all code changes are made in accordance with the cargo-clippy rules. 

However, here for better readability, we can add `assert_false!` macro (or use https://docs.rs/all_asserts/latest/all_asserts/macro.assert_false.html).

Reply:
`assert_false!` sounds good to me